cpp/tensorrt_llm/common/attentionOp.h

```diff
@@ -412,6 +412,23 @@ class AttentionOp
     // a plugin field or a constructor parameter
     int32_t mNbMultiBlockSemaphores = 0;
 
+    // [Chunked Attention]
+    // Chunked attention is applied to context requests only. Chunked attention will be
+    // applied when this field is specified and mMaskType == CAUSAL.
+    //
+    // In chunked attention, we break context requests into chunks of a specified size. Tokens can only
+    // attend to tokens in the same chunk. So, for example, if the chunk size is 3, we might have a mask
+    // that looks like this:
+    //
+    // 1 0 0 0 0 0
+    // 1 1 0 0 0 0
+    // 1 1 1 0 0 0
+    // 0 0 0 1 0 0
+    // 0 0 0 1 1 0
+    // 0 0 0 1 1 1
+    //
```

Comment:
Can you move this to `tensorrt_llm/_torch/modules/attention.py`? That file is more user facing.